web/source/osk/oskKey.ts

```diff
@@ -430,5 +430,19 @@ namespace com.keyman.osk {
 
       return t;
     }
+
+    public isUnderTouch(touch: Touch): boolean {
+      let x = touch.clientX;
+      let y = touch.clientY;
+
+      let btn = this.btn;
+      let x0 = dom.Utils.getAbsoluteX(btn); 
+      let y0 = dom.Utils.getAbsoluteY(btn);//-document.body.scrollTop;
+      
+      let x1=x0 + btn.offsetWidth;
+      let y1=y0 + btn.offsetHeight;
+
+      return (x > x0 && x < x1 && y > y0 && y < y1);
+    }
```

Comment:
In essence, this method computes the old `onKey` value within the loop seen in the block below, which had been removed by https://github.com/keymanapp/keyman/pull/5288/commits/063c326f5e0e088f85287129abbc20fbb3010820:

https://github.com/keymanapp/keyman/blob/d68fb7bac3edc50fe16c296459f61ee4dce6ac90/web/source/osk/visualKeyboard.ts#L2353-L2364

Turns out it had always been good logic; it's just that it wasn't being used at the time.  Note that `this.btn` is a perfect match for the loop's `sk`.